.gitignore

```diff
@@ -23,3 +23,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.idea
```

Comment:
The `.idea` folder should be ignored on your local git settings globally, if we had to ignore all folders/files from all editors in the repo it would be crazy to maintain. Using an editor is a personal choice, so the configuration should be local. So this should be removed.

Reply:
✅ Removed `.idea`